docs/docs/basics/getting-started.md

```diff
@@ -23,7 +33,14 @@ Then, you need to set up the player. This usually takes less than a second:
 ```ts
 import TrackPlayer from 'react-native-track-player';
 
-await TrackPlayer.setupPlayer()
+async function setupPlayer() {
+  await TrackPlayer.setupPlayer();
+  // The player is ready to be used
+}
+
+function App(): JSX.Element {
+  await setupPlayer();
```

Comment:
This isn't the proper way to call `setupPlayer` as you'll get an exception if `App` ever re-renders and `setupPlayer` has already been called. We should probably have a minimal example of what's happening in the `example` app and then link to it for more detailed information as there are a lot of ways this can be done.